packages/controls/src/widget_float.ts

```diff
@@ -237,6 +239,18 @@ export class FloatLogSliderView extends BaseIntSliderView {
     }
   }
 
+  /**
+   * Called when the slider handle is released after dragging,
+   * or by tapping or moving by the arrow keys.
+   */
+  handleSliderChangeEvent(values: number[], handle: number): void {
+    const base = this.model.get('base');
+    const actual_value = Math.pow(base, this._validate_slide_value(values[0]));
+    this.readout.textContent = this.valueToString(actual_value);
+
+    this.handleSliderChanged(values, handle);
```

Comment:
I guess one problem that can occur is that when you tap or move by the arrow keys, you will get two events: one from the update event, and one from the change event. If we wanted to avoid this, we could conditionally only pass the event on if `continuous_update` is false here (and other similar event handlers). I'm just not sure how bad of an issue this duplicate is, and if it can lead to some issues with multiple users, or if backbone will automatically discard the second change as a no-op.